examples/all-clusters-app/Ameba/README.md

```diff
@@ -0,0 +1,80 @@
+# CHIP Ameba All Clusters Example
+
+A prototype application that demonstrates device commissioning and cluster
+control.
+
+---
+
+-   [CHIP Ameba All Clusters Example](#chip-ameba-all-clusters-example)
+    -   [Supported Device](#supported-device)
+    -   [Building the Example Application](#building-the-example-application)
+    -   [Commissioning and cluster control](#commissioning-and-cluster-control)
+        -   [Commissioning](#commissioning)
+        -   [Cluster control](#cluster-control)
+
+---
+
+## Supported Device
+
+The CHIP demo application is supported on
+[Ameba RTL8722DM Board](https://www.amebaiot.com/en/amebad).
+
+## Building the Example Application
```

Comment:
Was this added to the vscode image?

We would like for developers to be able to run test builds without additional steps, at least when running the vscode image. 


I expect the following changes:

https://github.com/project-chip/connectedhomeip/blob/master/integrations/docker/images/chip-build-vscode/Dockerfile should have some copy of the ameba SDK readily available.

build_example.py scripts should be able to build this. Looking at
https://github.com/project-chip/connectedhomeip/blob/master/scripts/build/build/targets.py 
should be a good starting point on how to add a platform/board

Reply:
Hi @andy31415 ,
Thanks for the review, Ameba has yet to be added to vscode image.
I have read the two links you sent, but I am sorry that I am unbale to fully understand how to achieve the feature other than apply the standard changes on these files. 
Therefore, may I know which platforms can we take full reference and how can we test or verify the function after we applied the changes.
Many thank!

Reply:
Hi @andy31415, @woody-apple, @jmeg-sfy, 
Can I ask for more information or instructions about the Docker Build?
Is the ultimate goal to build the platforms' sdk using docker container instead of clone platform sdk to build?
I notice some platforms are using the docker build but some platforms seem still using the latter way.
Could you please suggest an sample example for us to refer to?
Also, as currently our sdk is built using a fixed makefile, could you please suggest how should we implement the build_example?
It will be so much appreciated if a rough porting process list can be provided.
Many thanks!